scripts/testacc_prepare_env.sh

```diff
@@ -19,11 +19,11 @@ if [[ -z "${ARGOCD_CI}" ]]; then
   kind load docker-image redis:6.2.4-alpine --name argocd
   kind load docker-image ghcr.io/dexidp/dex:v2.27.0 --name argocd
   kind load docker-image alpine:3 --name argocd
-  kind load docker-image quay.io/argoproj/argocd:${ARGOCD_VERSION:-v1.8.7} --name argocd
+  kind load docker-image quay.io/argoproj/argocd:${ARGOCD_VERSION:-v2.1.10} --name argocd
 fi
 
-echo "\n--- Install ArgoCD ${ARGOCD_VERSION:-v1.8.7}\n"
-curl https://raw.githubusercontent.com/argoproj/argo-cd/${ARGOCD_VERSION:-v1.8.7}/manifests/install.yaml > manifests/install/argocd.yml &&
+echo "\n--- Install ArgoCD ${ARGOCD_VERSION:-v2.1.10}\n"
```

Comment:
Let's drop support for 2.1.x, as per the compatibility promise, and let's raise this to the latest 2.3.x.

Reply:
Done.